src/components/backend-ai-indicator.ts

```diff
@@ -45,19 +46,25 @@ export default class BackendAIIndicator extends LitElement {
     super.connectedCallback();
   }
 
-  start(mode = 'determinate') {
+  async start(mode = 'determinate') {
     this.value = 0;
-    this.text = 'Initializing...';
     this.mode = mode;
+    await this.updateComplete;
     this.dialog.show();
   }
 
   set(value, text = '') {
     this.value = value / 100.0;
     this.text = text;
+    if (value >= 1) {
```

Comment:
Shouldn't it be `this.value`, not `value`?

Reply:
Fixed at 413c09ce498